boards/arm/stm32h747i_disco/stm32h747i_disco_m7_defconfig

```diff
@@ -43,3 +43,6 @@ CONFIG_SERIAL=y
 # console
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
+
+# enable SDMMC driver
+CONFIG_DISK_ACCESS_STM32_SDMMC=y
```

Comment:
Please remove this.
Flag will be enabled depending on sdmmc1 node activation and DISK_ACCESS activation by application

Reply:
But then driver source files is not included. Should I really remove it?

Reply:
This symbol will be set to 'y' if a node with "st,stm32-sdmmd" is set to `status = "okay";` and `CONFIG_DISK_ACCESS=y`.
So, it will be enabled when needed (ie requested by application).
We're doing the same will all other peripheral symbols (aside from those always enabled because always requested: SERIAL, CLOCK_CONTROL, INTERRUPT_CONTROLLER, PINMUX).

cf https://docs.zephyrproject.org/latest/guides/porting/board_porting.html#general-recommendations

Reply:
Please consider this one as the last blocking comment.

Reply:
OK, CONFIG_DISK_ACCESS removed, code synced with PLL3 changes